src/coreComponents/fileIO/timeHistory/PackCollection.cpp

```diff
@@ -200,42 +228,39 @@ void PackCollection::buildMetaCollectors( )
     {
       coordField = ElementSubRegionBase::viewKeyStruct::elementCenterString();
     }
-    else
-    {
-      GEOSX_ERROR( "Data collection for \"" << m_objectPath << "\" is unimplemented." );
-    }
+//    else
+//    {
+//      GEOSX_ERROR( "Data collection for \"" << m_objectPath << "\" is unimplemented." );
+//    }
 
-    string metaName( "coordinates" );
-    std::unique_ptr< PackCollection > coordCollector = std::make_unique< PackCollection >( metaName, this );
-    coordCollector->getWrapper< string >( PackCollection::viewKeysStruct::objectPathString() ).reference() = m_objectPath;
-    coordCollector->getWrapper< string >( PackCollection::viewKeysStruct::fieldNameString() ).reference() = string( coordField );
-    coordCollector->getWrapper< string_array >( PackCollection::viewKeysStruct::setNamesString() ).reference() = m_setNames;
-    coordCollector->getWrapper< localIndex >( PackCollection::viewKeysStruct::onlyOnSetChangeString() ).reference() = 1;
-    // don't recursively keep creating coordinate metacollectors
-    coordCollector->disableCoordCollection( );
-    m_metaCollectors.push_back( std::move( coordCollector ) );
+    string metaName( "coordinates" ); // FIXME Name?
+    std::unique_ptr< PackCollection > collector = std::make_unique< PackCollection >( metaName, this );
+    collector->m_objectPath = m_objectPath;
+    collector->m_fieldName = coordField ? string( coordField ) : m_fieldName;
```

Comment:
Not sure of what should happen in the case `coordField` stays a `nullptr`

Reply:
I still need to finish this kind of "details"!

Reply:
when `nullptr` is assigned to an actual pointer variable, it decays to the null value for that pointer type, in pretty much all cases 0, the implicit conversion of that to bool will be `false`, and nonzero value (including an uninitialized pointer var) will be true. `coordField != nullptr` (which would resolve to `coordField != (char const *)0`) would avoid the uninitialized case, though I doubt we'll ever actually run into that here.

Reply:
In this precise case, `coordField` is either defined to an existing value or explicitly set to `nullptr`.
So I guess that checking its value is valid. Am I correct?

Reply:
@TotoGaz Yeah since we initialize the variable in all cases it should never erroneously return `true`